localstack/services/events/events_listener.py

```diff
@@ -0,0 +1,67 @@
+import json
+import re
+import tempfile
+import uuid
+
+from requests.models import Response
+
+from localstack.constants import TEST_AWS_ACCOUNT_ID, MOTO_ACCOUNT_ID
+from localstack.services.generic_proxy import ProxyListener
+from localstack.utils.aws import aws_stack
+from localstack.utils.common import to_str, save_file, TMP_FILES, mkdir
+
+EVENTS_TMP_DIR = tempfile.gettempdir() + '/cw_events/'
+
+
+class ProxyListenerEvents(ProxyListener):
+
+    def forward_request(self, method, path, data, headers):
+        if method == 'POST' and path == '/' and headers['X-Amz-Target'] == 'AWSEvents.PutEvents':
```

Comment:
Better:
```
action = headers.get('X-Amz-Target')
if method == 'POST' and path == '/' and action == 'AWSEvents.PutEvents':
    ...
```
(to avoid `KeyError` if the `X-Amz-Target` header is not present)

Reply:
Good point, thanks.